helm/linkerd-bundle/templates/apps.yaml

```diff
@@ -0,0 +1,84 @@
+{{- range $key, $value := .Values.apps }}
```

Comment:
Not for today, but I think this would be a perfect candidate to be put into a Giant Swarm app helper library chart

Reply:
Not sure if it's worth, so far there are only 3 bundles (leaving aside CAPI default apps) and they all have a few differences.

wdyt about a template-bundle repo instead?